whatlies/transformers/_normalizer.py

```diff
@@ -1,18 +1,18 @@
-from sklearn.preprocessing import normalize
+from sklearn.preprocessing import normalize, FunctionTransformer
 
-from whatlies.transformers import Transformer
+from ._transformer import SklearnTransformer
 from whatlies import EmbeddingSet
 from whatlies.transformers._common import new_embedding_dict
 
 
-class Normalizer(Transformer):
+class Normalizer(SklearnTransformer):
```

Comment:
@koaning Was there any problem, since I see that you have chosen to not remove the `fit` and `transform` methods after subclassing `SklearnTransformer`?

Reply:
Well spotted! I'll check. I recall correcting that.